### PR TITLE
Allows discussions/show to render faster

### PR DESCRIPTION
**Pull request by gabrielpoca**
Allows discussions/show to render while a request is made to the backend,
updating the interface once it's finished. In the future it wil probably need
a loading indication somewhere.

### CODE REVIEW

File: app/assets/javascripts/app/core/models/discussion.js [hunk 1]
```diff
@@ -13,17 +13,10 @@
...
-            return $q(function(resolve, reject) {
-              DS.refresh('discussion', discussion.id);
-              resolve();
-            });
-          };
+          var refesh = angular.bind(this, this.refresh);
 
           return DS.create('discussionComment', comment)
-            .then(refresh);
+            .then(refesh);
```
Comment (joaojusto) [inline app/assets/javascripts/app/core/models/discussion.js]: refresh

File: app/assets/javascripts/app/core/models/discussion.js [hunk 2]
```diff
@@ -13,17 +13,10 @@
...
         addComment: function(comment) {
-          var discussion = this;
-
-          var refresh = function() {
-            return $q(function(resolve, reject) {
-              DS.refresh('discussion', discussion.id);
-              resolve();
-            });
-          };
+          var refesh = angular.bind(this, this.refresh);
```
Comment (joaojusto) [inline app/assets/javascripts/app/core/models/discussion.js]: :gb:

**Comment by gabrielpoca** [comment]
@joaojusto done

**Comment by joaojusto** [comment]
:+1:

**Pull request merged**